lib/modules/datasource/python-version/readme.md

```diff
@@ -0,0 +1,36 @@
+This datasource returns Python releases from the [python.org API](https://www.python.org/api/v2/downloads/release/).
+
+It also fetches deprecated versions from the [Endoflife Date datasource](/modules/datasource/endoflife-date/).
```

Comment:
```suggestion
It also fetches deprecated versions from the [Endoflife Date datasource](../endoflife-date/index.md).
```
use relative paths, otherwise it will fail on mkdocs generation.